test/Streamly/Test/Common/Array.hs

```diff
@@ -192,6 +208,8 @@ main =
             prop "length . fromStream === n" testLengthFromStream
             prop "toStream . fromStream === id" testFromStreamToStream
             prop "read . write === id" testFoldUnfold
+            prop "First N elements of a list" testFromListN
```

Comment:
You'll need to wrap this around applicable `#if`

Reply:
This is applicable for ALL so need not to wrap with #if